srml/contract/src/wasm/runtime.rs

```diff
@@ -35,6 +35,22 @@ enum SpecialTrap {
 	Return(OutputBuf),
 }
 
+/// Contract status codes, intended to embed within `sandbox::HostError`
+/// Negative status codes are mapped from the WASM sandbox types (see `sandbox::Error`)
+#[allow(dead_code)]
+enum StatusCode {
+	InvalidMemoryAccess = -4, // Invalid memory access, likely outside user control
+	InvalidModule = -3, // Contract wasm module is invalid
+	BadExport = -2, // Failed to execute an exported function
+	Internal = -1, // Generic internal error
+	Success = 0, // Generic success status
+	Failed = 1, // Generic failure status
```

Comment:
Same question as for `Internal`, plus, if still relevant, what is the difference between these two?